packages/@aws-cdk/aws-efs/test/integ.efs.expected.json

```diff
@@ -453,6 +453,7 @@
     "FileSystem8A8E25C0": {
       "Type": "AWS::EFS::FileSystem",
       "Properties": {
+        "Encrypted": true,
```

Comment:
This changed because our integ tests automatically enable all feature flags. Existing projects won't be affected (covered by the unit test).

https://github.com/aws/aws-cdk/blob/8782e672d6a8f8bbe201f2572c4b0fca7589168d/tools/cdk-integ-tools/lib/integ-helpers.ts#L341-L342